azure-client-runtime/src/main/java/com/microsoft/azure/AzureClient.java

```diff
@@ -232,7 +233,6 @@ public void setLongRunningOperationRetryTimeout(int longRunningOperationRetryTim
         pollingState.withResourceType(resourceType);
         pollingState.withSerializerAdapter(restClient().serializerAdapter());
         return Observable.just(true)
-                .subscribeOn(Schedulers.io())
```

Comment:
Why aren't we running this on the IO scheduler anymore?

Reply:
I think I just merged master in and this work happened to come along.
https://github.com/Azure/autorest-clientruntime-for-java/pull/190